modules/controlpanel.cpp

```diff
@@ -442,6 +446,10 @@ class CAdminMod : public CModule {
             bool b = sValue.ToBool();
             pUser->SetTimestampAppend(b);
             PutModule("AppendTimestamp = " + CString(b));
+        } else if (sVar == "builtinauthdisabled") {
+            bool b = sValue.ToBool();
+            pUser->SetBuiltinAuthDisabled(b);
+            PutModule("BuiltinAuthDisabled = " + CString(b));
```

Comment:
Is it a good idea for users to be able to edit this setting for themselves? I'd think that only admin should be able to change this setting.